easybuild/tools/job/gc3pie.py

```diff
@@ -96,9 +93,19 @@ def init(self):
 
         Start a new list of submitted jobs.
         """
-        self.output_dir = os.path.join(os.getcwd(), 'easybuild-gc3pie-jobs')
+        cfgfile = build_option('job_backend_config')
+        if cfgfile:
+            # FIXME: is there a better way?
```

Comment:
It will work for normal users; it does not work in the test suite, because of the `reload()` issue.  But normal usage will be just fine.